### ISSUE TITLE
[Bug]: Error loading script: !adetailer.py. ImportError: cannot import name 'data_path' from 'modules.paths'.

### ISSUE THREAD

**Issue by tatwork2** [Bing-su/adetailer#183]
### Describe the bug

Error loading script: !adetailer.py
Traceback (most recent call last):
File "B:\SD4\sd.webui\webui\modules\scripts.py", line 218, in load_scripts
script_module = script_loading.load_module(scriptfile.path)
File "B:\SD4\sd.webui\webui\modules\script_loading.py", line 13, in load_module
exec(compiled, module.__dict__)
File "B:\SD4\sd.webui\webui\extensions\adetailer\scripts\!adetailer.py", line 31, in <module>
from adetailer.ui import adui, ordinal, suffix
File "B:\SD4\sd.webui\webui\extensions\adetailer\adetailer\ui.py", line 11, in <module>
from controlnet_ext import controlnet_exists, get_cn_models
File "B:\SD4\sd.webui\webui\extensions\adetailer\controlnet_ext\__init__.py", line 1, in <module>
from .controlnet_ext import ControlNetExt, controlnet_exists, get_cn_models
File "B:\SD4\sd.webui\webui\extensions\adetailer\controlnet_ext\controlnet_ext.py", line 9, in <module>
from modules.paths import data_path, models_path, script_path
ImportError: cannot import name 'data_path' from 'modules.paths' (B:\SD4\sd.webui\webui\modules\paths.py)

### The entire console log, including python version information, webui version, commit hash, commandline arguments and error traces.

```Shell
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]                                                                                           Commit hash: ff74baceb717ab78264aee30fc816de1effa7b5c
Launching Web UI with arguments: --xformers --medvram --autolaunch
```

**Comment by Bing-su** [comment]
Try reinstalling

**Comment by tatwork2** [comment]
thanks, it worked

**Comment by haythamodeh** [comment]
Im getting this error when loading adetailer:
Error loading script: !adetailer.py
Traceback (most recent call last):
File "/Users/-----/Desktop/Stable Diffusion Web UI.app/Contents/Resources/stable-diffusion-webui/modules/scripts.py", line 382, in load_scripts
script_module = script_loading.load_module(scriptfile.path)
File "/Users/-----/Desktop/Stable Diffusion Web UI.app/Contents/Resources/stable-diffusion-webui/modules/script_loading.py", line 10, in load_module
module_spec.loader.exec_module(module)
File "", line 883, in exec_module
File "", line 241, in _call_with_frames_removed
File "/Users/-----/Documents/Stable Diffusion Web UI/extensions/adetailer/scripts/!adetailer.py", line 17, in
from rich import print
ModuleNotFoundError: No module named 'rich'

I tried installing rich with pip, still didn't work, I also tried updating requirements.txt and requirements_versions.txt.
i tried deleting venv folder, but got stuck with some issues so I restored the venv back, I'm not sure why this problem is happening. any help would be appreciated. thanks
